nanomi-optics/nanomi_optics/gui/window_main.py

```diff
@@ -71,6 +73,12 @@ def __init__(self):
         self.current_lens = -1
         self.last_lens = -1
 
+        self.save = tk.Button(self, text="Save", command=self.save_results)
+        self.save.grid(row=3, column=0, sticky="nwse")
+        settings_frame
+        self.reset = tk.Button(self, text="Reset", command=self.reset_settings)
+        self.reset.grid(row=3, column=1, sticky="nwse")
```

Comment:
also, personal preference, but think save button on the right side feels more intuitive.